hbase-client/src/main/java/org/apache/hadoop/hbase/client/SimpleRequestController.java

```diff
@@ -433,7 +433,7 @@ private void waitForRegion() throws InterruptedIOException {
      *
      * @param loc
      * @param heapSizeOfRow
-     * @return
+     * @return either Include {@link ReturnCode} or Skip {@link ReturnCode}
```

Comment:
Another option if the parameter or return code are self-explanatory is to just remove them from javadoc. Perhaps not here but in the below where you add the text 'the row key' as description on a param named 'key'...